### ISSUE TITLE
Unwanted arrow in Mega Menu

### ISSUE THREAD

**Issue by KR1275** [templaza/astroid-framework#608]
Hello,

Since the last update I see an arrow in the megamenus. 
<img width="265" alt="arrowmm" src="https://github.com/templaza/astroid-framework/assets/163830515/0eaff8a0-61d0-4912-801d-8667a9715fb9">

How do I remove this? I didn't find any new setting.

**Comment by KR1275** [comment]
Yes, I use the right menu-item. It worked and it still works fine. It's just that there is an arrow since the last Astroid update and I want to remove it.

**Comment by Chacapamac** [comment]
It is weird as I have a mega menu and no Arrow. I guess you will have to find the element on your web browser inspector and hide it with an Custom CSS rule. Be sure to put the right CSS selector to hide only that specific arrow.

“element”  {display: none;}

**Comment by KR1275** [comment]
Thanks. I think it's gone now. Astroid changed the setting in my menus (from level 1 to 1, was changed into from 1 to endless).

**Comment by SoundHunter1** [comment]
Hi @KR1275,

Dit you make these changes in the template settings or in the menu item (if possible)?
At my end they are still visible and I do have the template set from 1 to all (but if I reduce to 1 than the submenu items disappear of course).
I can't find the reason the arrows are visible but indeed it looks something started with the latest version.

Thank you!

**Comment by KR1275** [comment]
I changed it in the template setting (not in the Joomla menu setting)

**Comment by SoundHunter1** [comment]
I still haven't find out why they keep visible at my side.
It is the fa-chevron-right class that creates the arrow to the right.
When I change the template setting under Header --> Site menu --> End level to 2 the menu items disappear, if I set it to 3 the menu items are back but also the arrows.

Do I overlook something?

**Comment by felixkat** [comment]
Closing as original issue resolved.
